src/View/Components/Alert.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace WireUi\View\Components;
+
+use Illuminate\Support\Str;
+use Illuminate\View\Component;
+
+class Alert extends Component
+{
+    public ?string $padding;
+
+    public ?string $rounded;
+
+    public bool $border;
+
+    public bool $shadow;
+
+    public bool $info;
+
+    public bool $warning;
+
+    public bool $success;
+
+    public bool $danger;
+
+    public ?string $icon;
+
+    public ?string $heading;
+
+    public ?string $text;
+
+    public ?string $actions;
+
+    public bool $dismiss;
+
+    public ?string $alertClasses = '';
+
+    public string $backgroundColor = 'bg-gray-50 dark:bg-secondary-700';
+
+    public string $subjectColor = 'text-gray-700 dark:text-gray-400';
+
+    public function __construct(
+        ?string $padding = 'p-4',
+        ?string $rounded = 'rounded-md',
+        ?string $icon = null,
+        ?string $heading = null,
+        ?string $text = null,
+        ?string $actions = null,
+        bool $border = false,
+        bool $shadow = false,
+        bool $info = false,
+        bool $warning = false,
+        bool $success = false,
+        bool $danger = false,
```

Comment:
```suggestion
        bool $error = false,
```

Reply:
Updated it to be more in line with the buttons
public bool $info = false,
public bool $warning = false,
public bool $positive = false,
public bool $negative = false,